### PR TITLE
TicketCounter

### LINKED ISSUE #109: Add ticket counters
Do we have an api to eventbrite to have a counter of the tickets still available in-person?

### PR DESCRIPTION
**Pull request by nicpuppa**
Closes #109

### CODE REVIEW

**Comment by nicpuppa** [comment]
@TheJoin95 @404answernotfound @Wabri @davideimola which is the best way to handle the OAuth2 token for you ?

**Comment by Wabri** [comment]
Being a quite important secret I would say to use netlify's internal vault if it exists

**Comment by davideimola** [comment]
I think you can set environment variables on Netlify

**Comment by TheJoin95** [comment]
On netlify we can use functions, edge functions or env variables

**Comment by 404answernotfound** [comment]
I'd go for netlify as well!

**Comment by nicpuppa** [comment]
@TheJoin95 @404answernotfound can we talk about Netlify functions solution in sync way organizing a meet ?

**Comment by TheJoin95** [comment]
Today I can't, I'm full all day, sorry. But I can eventually add the values

Il mar 13 dic 2022, 12:02 Nicola Puppa ***@***.***> ha
scritto:

File: components/Hero.tsx [hunk 1]
```diff
@@ -100,6 +102,14 @@ const Hero = ({
           </a>
         )}
       </div>
+
+      <br />
+
+      <div className='cta-container'>
+          {props.showTicketAvailability && (
+            <TicketCounter />
```
Comment (404answernotfound) [inline components/Hero.tsx:110]: I like this and I think we can already merge but can we also try to have the tickets on the right side of the hero?

**Review by 404answernotfound — APPROVED** [review]

File: components/Hero.tsx (hunk 1 above)
Comment (nicpuppa) [reply components/Hero.tsx:110]: I merge this, we try to move this component in another pr

**Pull request merged**